kitsune/products/migrations/0005_product_parent.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.22 on 2019-09-23 05:52
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('products', '0004_product_codename'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='product',
+            name='parent',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='subproducts', to='products.Product'),
```

Comment:
Do we want to delete the children if the parent gets deleted ?

Reply:
It's a logical assumption that we do if we just drop the parent without nullyfing it first. That said, I will double check this